src/edit.js

```diff
@@ -273,6 +275,48 @@ function Edit( props ) {
 						checked={ preview }
 						onChange={ () => setAttributes( { preview: ! preview } ) }
 					/>
+					<ToggleGroupControl
+						label={ __( 'EQUALIZER', 'winamp-block' ) }
+						onChange={(currentPosEqu) => {
+								setAttributes({ currentPosEqu });
+								//console.log("currentPosEqu: " + currentPosEqu.toString());
+							}
+						}
+						value={ currentPosEqu }
+					>
+						<ToggleGroupControlOption label={ __( 'Top', 'winamp-block' ) } value="1" />
+						<ToggleGroupControlOption label={ __( 'Middle', 'winamp-block' ) } value="2" />
+						<ToggleGroupControlOption label={ __( 'Bottom', 'winamp-block' ) } value="3" />
+						<ToggleGroupControlOption label={ __( 'Hide', 'winamp-block' ) } value="0" />
```

Comment:
The `value` attributes should have meaningful values such as `middle`, `hide` etc instead of numbers so that we it is easily readable.